webrender/src/frame_builder.rs

```diff
@@ -951,6 +951,7 @@ impl FrameBuilder {
                 }
             }
             BoxShadowKind::Shadow(rects) => {
+                assert!(blur_radius > 0.0);
```

Comment:
It will fire if both `blur_radius == 0.0` and `border_radius == 0.0`. Where is this case handled?